data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,366 @@
+			`	Given the dangers of Unfettered space, you figure it would be a good idea to familiarize yourself with the design, and maybe to modify it.`
+	on enter
+		system
+			attributes "unfettered"
+		"old reputation: Hai (Unfettered)" = "reputation: Hai (Unfettered)"
+		"reputation: Hai (Unfettered)" >?= 0
+		dialog `As you enter the system, you prepare for a fight to get the cargo through, but, surprisingly, it is very quiet this time.`
+			`	Indeed, for the first time, entering Unfettered space does not mean getting shot at, potentially by missiles all sides.`
+			`	They are not hailing you either. Perhaps they think you are one of the Hai aiding them, especially given your cargo and the ship you are flying.`
+	on fail
+		"reputation: Hai (Unfettered)" <?= -1000
+		dialog `The Unfettered and their associates are not clearly happy when you tell them about this. "You will have to prove yourself another way."`
+	on visit
+		dialog `You have reached <planet>, but you either did not bring the Ancient Ladybug, some of the cargo was on another ship, or you are not using it as your flagship.`
+	to complete
+		has "flagship model: Ancient Ladybug"
+	on complete
+		dialog `You receive a message from the 'associates of Hai-home', "thank you for helping our brethren, <last>. We will not forget this. Please accept these <payment> to get you started on paying back."`
+		payment 100000
+		log "Delivered a pressing food cargo to the Unfettered using a newly acquired but pretty old Ladybug. They seem to not be firing on me, for now at least."
+
+
+
+# no category, meaning you can't remove it from the Ladybug
+outfit "Unfettered Landing Permit"
+	"maintenance costs" 681
+	description "This item is needed in order to land on Unfettered planets, and its loss will cancel the arrangement."
+
+mission "Hai Legacy 3"
+	name "Relic of a long lost era"
+	description "Beat the Lightning Bugs in a merciless fight. You only have one day."
+	landing
+	source "Firelode"
+	deadline 2
+	to offer
+		has "flagship model: Ancient Ladybug"
+		has "Hai Legacy 2: done"
+	on offer
+		conversation
+			`As you land and get out of your ship, you quickly notice you have attracted a lot of attention, probably due to this old design. One Unfettered that looks like an engineer comes to speak to you, warmer than usual. "Welcome. I do not know how you have obtained this ship, and some will probably see it as a sign. As you may know it is a relic of our past, a past we are quite proud of."`
+			`	"In truth, I do not personally care about all of that. All I care about is that this ship is old, and has technology that would be very valuable because of how rare it is. Thus, I challenge you for it!"`
+			choice
+				`	"What do I get if I win?"`
+					goto win
+				`	"Can't I just sell it to you?"`
+			`	"No, because in the eyes of my brethren that would mean I did not deserve this ship."`
```

Comment:
This premise could use improving -> if they win you are likely dead, which will make acquiring the technology on the ship rather difficult. I recommend a different excuse for this fight.

Reply:
Well their ship is present to monitor it
Mb they need to prove their worth as they are starting captain? True reason is they'd capture the ship but I cant do that rn

Reply:
I also did not want to trouble the player with not killing them